server/src/controller_test.py

```diff
@@ -195,4 +198,3 @@ def test_ok():
 def test_error():
     assert isinstance(controller.error('oops'), dict)
     assert controller.error('oops') == {'ok': False, 'message': 'oops'}
-
```

Comment:
It's actually better to keep these lines at the end. It helps version control systems.

Reply:
One empty line is enough, no?